lib/runtimeSettings.js

```diff
@@ -66,6 +66,9 @@ function getSettingsFile (settings) {
         if (settings.settings.disableEditor !== undefined) {
             projectSettings.disableEditor = `disableEditor: ${settings.settings.disableEditor},`
         }
+        if (settings.settings.ha?.replicas >= 2) {
+            projectSettings.disableEditor = 'disableEditor: true'
```

Comment:
```suggestion
            projectSettings.disableEditor = 'disableEditor: true,'
```

Reply:
The `,` is needed, this is building a string, not setting a variable.